hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestBackupSystemTable.java

```diff
@@ -485,15 +485,12 @@ private BackupInfo createBackupInfo() {
     return ctxt;
   }
 
-  private List<BackupInfo> createBackupInfoList(int size) {
+  private List<BackupInfo> createBackupInfoList(int size) throws InterruptedException {
     List<BackupInfo> list = new ArrayList<>();
     for (int i = 0; i < size; i++) {
       list.add(createBackupInfo());
-      try {
-        Thread.sleep(10);
-      } catch (InterruptedException e) {
-        e.printStackTrace();
-      }
+      // XXX Why do we need this sleep?
+      Thread.sleep(100);
```

Comment:
It is 10 ms, not 100 ms?

Reply:
10ms is really short dont' you think? But it's fair that this patch shouldn't try to do more, will change it back.